packages/data-context/src/actions/WizardActions.ts

```diff
@@ -312,18 +432,27 @@ export class WizardActions {
   }
 
   private async scaffoldFile (filePath: string, contents: string, description: string): Promise<NexusGenObjects['ScaffoldedFile']> {
-    if (fs.existsSync(filePath)) {
+    try {
+      await this.ctx.fs.stat(filePath)
```

Comment:
Isn't there a way to check if the file exists that DOESNT error (eg, returns null)? This try/empty catch seems a bit heavy handed.

Reply:
You are right we should probably follow the steps here https://gist.github.com/joepie91/bbf495e044da043de2ba
I will update